### ISSUE TITLE
quest bug saving yenniku

### ISSUE THREAD

**Issue by swetchment** [elysium-project/classic-bug-tracker#1313]
The last part of the quest is to fill the soul gem of yenniku. When stunning him, he doesn't give filled soul gem afterward. He doesn't do anything.. please fix

**Comment by Huntitude** [comment]
Yes, please fix this. Almost been 2 weeks. Such good reward from this quest and its bugged.

Quest: Saving Yenniku (stv)
Objective: Use Soul Gem (provided) on Yenniku to and he becomes friendly. 
Issue: He should be able to talk to after you use the gem, but he doesn't have the interactive option.
